ci/properties/laravel.properties.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Laravel",
+    "description": "Test a Laravel project.",
+    "iconName": "php",
+    "categories": [
+        "Php",
```

Comment:
Should this be `PHP`?

Reply:
In categories I have PHP now.
https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa